composer.json

```diff
@@ -146,7 +146,7 @@
         "symfony/flex": "^1.7",
         "symfony/web-profiler-bundle": "^4.4",
         "symplify/package-builder": "^8.0 <8.3",
-        "vimeo/psalm": "4.1.1"
+        "vimeo/psalm": "^4.3"
```

Comment:
We should leave this constraint to point to a specific version of Psalm so that updates don't break our builds.